cmd/controller-manager/main.go

```diff
@@ -316,23 +317,75 @@ func addMonitoringKeyLabelToOperatorService(ctx context.Context, cfg *rest.Confi
 }
 
 func installMonitoring(ctx context.Context, client client.Client) error {
-	log.Info("Installing monitoring resources")
 	params := map[string]string{"Namespace": os.Getenv("NAMESPACE")}
-
 	templateHelper := util.NewTemplateHelper(params)
 
 	for _, template := range templateHelper.TemplateList {
 		resource, err := templateHelper.CreateResource(template)
 		if err != nil {
-			return fmt.Errorf("createResource failed: %s", err)
+			return fmt.Errorf("DEBUG createResource failed: %s", err)
```

Comment:
```suggestion
			return fmt.Errorf("createResource failed: %s", err)
```

I presume this change is unwanted.